main.cpp

```diff
@@ -498,13 +559,17 @@ void BayernMunich::celebrateGoal(std::string starPlayer)
 int main()
 {
     Car myBrothersCar;
-    myBrothersCar.predictRaceWinProbability("Ford", 250);
+    bool brotherWinProbability = myBrothersCar.predictRaceWinProbability("Ford", 250);
+    std::cout << "Brother win probability " << brotherWinProbability << std::endl;
+    myBrothersCar.printPredictRaceWinProbability();
     float brothersCarSellValue = myBrothersCar.calculateTrueSellValue("Ford", 6000.75f, 100000);
     std::cout << "True sell value of brothers car: $" << brothersCarSellValue << std::endl; 
+    myBrothersCar.printCalculateTrueSellValue();
     myBrothersCar.calculateMilesUntilOilChange(1, 10000);
-    
+
     Car mySistersCar;
-    mySistersCar.predictRaceWinProbability("Dodge");
+    bool sisterWinProbability = mySistersCar.predictRaceWinProbability("Dodge");
+    std::cout << "Sister win probability " << sisterWinProbability << std::endl;
     float sistersCarSellValue = mySistersCar.calculateTrueSellValue("Dodge", 5000.75f, 10000);
```

Comment:
use `auto` in case you decide to change the function's return type later
- [ ] revise